sympy/solvers/solvers.py

```diff
@@ -124,7 +126,8 @@ def _simple_dens(f, symbols):
 
 
 def denoms(eq, *symbols):
-    """Return (recursively) set of all denominators that appear in eq
+    """
+    Return (recursively) set of all denominators that appear in *eq*
     that contain any symbol in ``symbols``; if ``symbols`` are not
```

Comment:
Should symbols here be in asterisks like eq?

Reply:
Yes there were a few places where I wasn't sure if symbols was referring to a parameter or code or should be in plain text, so let me know what you think.

Reply:
Looks like here it is referring to the parameter.